examples/sveltekit-pwa/src/routes/index.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+    import Counter from '$lib/components/Counter.svelte'
+    import Go from '$lib/components/Go.svelte'
+</script>
+
+<Counter />
+
+<br/>
+
+<Go />
+
+<br />
+
+<a href="/about">About</a>
+
+<br />
```

Comment:
just personal preference, but I find this a bit hard to read with so many blank lines
```suggestion
<Counter /><br/>

<Go /><br />

<a href="/about">About</a><br />
```